packages/edge-gateway/src/durable-objects/summary-metrics.js

```diff
@@ -109,13 +120,15 @@ export class SummaryMetrics0 {
     switch (url.pathname) {
       case '/metrics/winner':
         await this._updateWinnerMetrics(data)
-        return new Response()
+        break
       case '/metrics/cache':
         await this._updatedCacheMetrics(data)
-        return new Response()
+        break
       default:
-        return new Response('Not found', { status: 404 })
+        throw new Error('Not found')
```

Comment:
How come this change? I expect a 500 to be returned in the response now but 404 was more accurate.